src/view/hook/useLocalStorage.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
```

Comment:
I think we should have a "top" level hooks folder (i.e. `src/hooks`) and move this there.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3792/commits/d7e48d30b6480d3af9083b3aa36eebb4331a2d3f